packages/xo-web/src/common/intl/messages.js

```diff
@@ -2243,6 +2243,7 @@ const messages = {
   auditSaveFingerprintInErrorInfo:
     'However, if you trust the current state of the records, keep this fingerprint to be able to check their integrity later.',
   auditNewFingerprint: 'New fingerprint',
+  downloadAuditRecords: 'Download records',
```

Comment:
"Export"?

Reply:
IMHO, `Download` is more intuitive for user than `Export`.This term is also used for `config`.